app/controllers/profile_controller.rb

```diff
@@ -9,6 +11,30 @@ def show
     @user = current_user
   end
 
+  def change_password
+    passwd_form = params[:password_change]
+    if passwd_form['new_password1'] != passwd_form['new_password2']
+      flash[:notice] = t('profile.password_mismatch')
+    else
+      backend_change_password(
+        old: passwd_form['old_password'],
+        proposed: passwd_form['new_password1'],
```

Comment:
```suggestion
        new_password: passwd_form['new_password1'],
```
for the sake of abstraction and readability... I know the cognito param is called `proposed` but we don't need to expose that and can make it nicer `old_password` and `new_password`